src/screens/PaymentDetailsScreen.tsx

```
@@ -5,25 +1,98 @@
+  useEffect(() => {
+    // Populate form when initial values change (e.g., when editing)
+    setPaymentTitle(initialTitle);
+    setPaymentShares(
+      new Map(Array.from(initialShares).map(([key, value]) => [key, value.toString()]))
+    );
+  }, [initialTitle, initialShares]);
+
+  const handleSave = () => {
+    const parsedShares = new Map<string, number>();
+    for (const [key, value] of paymentShares) {
+      const parsedValue = parseFloat(value);
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)